tests/unit/test_cli.py

```diff
@@ -128,6 +128,18 @@ def test_cli_gui_visibility(monkeypatch, trogon_installed):
             ],
             "nipoppy.workflows.bids_conversion.BidsConversionRunner",
         ),
+        (
+            [
+                "bidsify",
+                "--dataset",
+                "[mocked_dir]",
+                "--pipeline",
+                "my_pipeline",
+                "--use-list",
+                "[mocked_file]",
+            ],
+            "nipoppy.workflows.bids_conversion.BidsConversionRunner",
+        ),
```

Comment:
Not sure if it's necessary. We don't test individual flags for other sub-commands right now. This could be something we can consider in the future?